src/components/ArticleCommentsList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useParams } from "react-router-dom";
+import { useState } from "react";
+import { fetchCommentsByArticleId } from "../api";
+import { useEffect } from "react";
+import { CommentCard } from "./CommentCard";
+
+export const ArticleCommentsList = () => {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState("");
+
+  useEffect(() => {
+    fetchCommentsByArticleId(article_id).then((data) => {
+    console
+      setComments(data)
+      console.log(comments, "setted comments")
+    });
+  }, [article_id]);
+
+  return (
+      comments && (
+          <ul className='comments-list'>
+              {comments.map((comment) => (
+                  <CommentCard comment={comment} key={comment.comment_id} />
+              ))}
+          </ul>
```

Comment:
Nice 👍 but please add some isLoading functionality as you may have to wait for the fetch to complete :)